app/models/manageiq/providers/vmware/infra_manager/refresh_parser.rb

```diff
@@ -368,10 +403,33 @@ def self.host_inv_to_switch_hashes(inv)
 
           pnics.each { |pnic| result_uids[:pnic_id][pnic] = new_result unless pnic.blank? }
         end
+
+        dvswitch_inv.to_miq_a.each do |data|
+          config = data.fetch('config', {})
+          uid = data['MOR']
+          security_policy = config.fetch('defaultPortConfig', {}).fetch('securityPolicy', {})
+
+          dvswitch_uid_ems[uid] || dvswitch_uid_ems[uid] = {
+            :uid_ems           => uid,
+            :name              => config['name'] || data.fetch_path('summary', 'name'),
+            :ports             => config['numPorts'] || 0,
+
+            :allow_promiscuous => security_policy.fetch_path('allowPromiscuous', 'value').to_s.casecmp('true') == 0,
+            :forged_transmits  => security_policy.fetch_path('forgedTransmits', 'value').to_s.casecmp('true') == 0,
+            :mac_changes       => security_policy.fetch_path('macChanges', 'value').to_s.casecmp('true') == 0,
+
```

Comment:
@jameswnl for standard switches if that isn't set we leave these fields nil
https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/vmware/infra_manager/refresh_parser.rb#L359-L361
I think we should stay consistent with how we handle the same fields on standard switches.